mongo-docker/Dockerfile

```diff
@@ -1,18 +1,18 @@
-# Based roughly around official MongoDB docker image
-# https://github.com/docker-library/mongo/blob/master/2.6/Dockerfile
-FROM alpine:3.2
+FROM mhart/alpine-node:0.10
 
-# Use a custom build script instead of messy chained together RUN
-# or multiple RUN statements that add bloat to the image
-ADD run.sh /tmp/run.sh
+ENV NODE_ENV=production
+
+COPY mongo-docker /tmp
```

Comment:
This `Dockerfile` is in the `mongo-docker` directory.  I believe all `COPY` commands are relative to the location of the `Dockerfile`.  Does this work?